src/itwinai/loggers.py

```diff
@@ -240,7 +240,7 @@ def serialize(self, obj: Any, identifier: str) -> str:
         Returns:
             str: local path of the serialized object to be logged.
         """
-        itm_path = os.path.join(self.savedir, identifier)
+        itm_path = Path(self.savedir) / str(identifier)
```

Comment:
If `self.savedir` is already a Path, shouldn't `self.savedir / str(identifier)` be sufficient here?